### PR TITLE
Problem: cometbft not up to date

### PR DESCRIPTION
**Pull request by yihuang**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 * (e2ee)[#1421](https://github.com/crypto-org-chain/cronos/pull/1421) Validate e2ee key when register.
 * [#1437](https://github.com/crypto-org-chain/cronos/pull/1437) Update cometbft and cosmos-sdk dependencies.
 * (rpc) [#1467](https://github.com/crypto-org-chain/cronos/pull/1467) Avoid unnecessary tx decode in tx listener.
+* [#]() Update cometbft to v0.37.7.
```
Comment (yihuang) [inline CHANGELOG.md:16]:
```suggestion
* [#1489](https://github.com/crypto-org-chain/cronos/pull/1489) Update cometbft to v0.37.7.
```

Comment (mmsqe) [inline CHANGELOG.md:16]: https://github.com/cometbft/cometbft/compare/8a0f4814d1dc...v0.37.7
```suggestion
* [#1489](https://github.com/crypto-org-chain/cronos/pull/1489) Update cometbft to v0.37.7.
```

**Review by mmsqe — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -13,8 +13,7 @@ require (
 	filippo.io/age v1.1.1
 	github.com/99designs/keyring v1.2.1
 	github.com/armon/go-metrics v0.4.1
-	// v0.37.x
-	github.com/cometbft/cometbft v0.37.7-0.20240515092623-8a0f4814d1dc
+	github.com/cometbft/cometbft v0.37.7
```
Comment (yihuang) [inline go.mod:16]: https://github.com/cometbft/cometbft/compare/8a0f4814d1dc...v0.37.7

**Comment by mmsqe** [comment]
seems no space left on device in upgrade

**Pull request merged**